server/providers/uploads/local/index.ts

```diff
@@ -0,0 +1,72 @@
+import { Provider } from '../../../uploads/types'
+import process from 'process'
+import path from 'path'
+import fs from 'fs'
+import crypto from 'crypto'
+import express from 'express'
+import config from '../../../../config/server'
+
+interface LocalProviderOptions {
+  uploadDirectory?: string;
+  endpoint: string;
+}
+
+interface Upload {
+  filePath: string;
+  name: string;
+}
+
+export default class LocalProvider implements Provider {
+  private uploadDirectory: string
+  private endpoint: string
+  private app: express.Application
+
+  private uploadMap: Map<string, Upload>
+
+  constructor (options: LocalProviderOptions, app: express.Application) {
+    if (options.uploadDirectory === undefined) {
+      options.uploadDirectory = path.join(process.cwd(), 'uploads')
+    }
+
+    if (!fs.existsSync(options.uploadDirectory)) {
+      fs.mkdirSync(options.uploadDirectory)
```

Comment:
We should either use the `mkdirp` module or just not create the dir at all.

Reply:
This would support the default installation of rctf where the uploads directory is just `/app/uploads`. I don't think we need to support weird custom configurations where the directory is nested somewhere deep.